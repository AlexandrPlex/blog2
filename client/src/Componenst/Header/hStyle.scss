@import url('https://fonts.googleapis.com/css?family=Roboto:400i|Titillium+Web:300,400,700');
$backGroudColor: #cbcbcb;
.hStyle{
  background-color: $backGroudColor;
  background: $backGroudColor; /* Для старых браузров */
  background: linear-gradient(to top, $backGroudColor, #4823f1);
  padding: 10px;
  font-family: 'Titillium Web', sans-serif;
  font-family: 'Roboto', sans-serif;
  display: -webkit-flex; /* Safari */
  display: flex;
}
