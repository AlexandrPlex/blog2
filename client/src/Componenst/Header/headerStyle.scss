@import url('https://fonts.googleapis.com/css?family=Roboto:400i|Titillium+Web:300,400,700');
@import "../../config";
.headerStyle{
  $gradient_0: #4460f1;
  $gradient_1: #000000;
  $logoColor: #ffffff;
  $font:'Titillium Web', sans-serif;
  background: $mainBackground; /* Для старых браузров */
  background: radial-gradient($gradient_0, $gradient_1);
  padding: 10px;
  overflow: hidden;
  display: -webkit-flex; /* Safari */
  display: flex;
  .logo{
    color: $logoColor;
    font-family: $font;
    margin-top: 0px;
  }
}
