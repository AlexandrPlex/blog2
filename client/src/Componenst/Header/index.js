// @flow

import React from 'react';
import './hStyle.scss';

export default class Header extends React.PureComponent<{}> {
    render () {
        return <div className='hStyle'>
            <h1>adsfkl</h1>
        </div>;
    }
}
