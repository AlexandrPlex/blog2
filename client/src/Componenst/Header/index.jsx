// @flow

import React from 'react';
import './headerStyle.scss';

export default class Header extends React.PureComponent<{}> {
    render () {
        return <div className='headerStyle'>
            <h1 className='logo'>Aplex for creating</h1>
        </div>;
    }
}
