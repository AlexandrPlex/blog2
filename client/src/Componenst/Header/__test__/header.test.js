import React from 'react';
import renderer from 'react-test-renderer';
import Header from '../index';

it('Header Snaphot', () => {
    const tree = renderer
        .create(<Header/>)
        .toJSON();
    expect(tree).toMatchSnapshot();
});
