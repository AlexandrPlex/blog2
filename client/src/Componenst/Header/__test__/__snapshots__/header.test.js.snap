// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Snaphot 1`] = `
<div
  className="headerStyle"
>
  <h1
    className="logo"
  >
    Aplex for creating
  </h1>
</div>
`;

exports[`renders correctly 1`] = `
<div
  className="headerStyle"
>
  <h1
    className="logo"
  >
    Aplex for creating
  </h1>
</div>
`;
